### ISSUE TITLE
[Bug]: Cannot unblock admin of an instance

### ISSUE THREAD

**Issue by phinas** [LemmyNet/lemmy-ui#1481]
### Requirements

- [X] Is this a bug report? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a single bug? Do not put multiple bugs in one issue.
- [ ] Is this a server side (not related to the UI) issue? Use the [Lemmy back end](https://github.com/LemmyNet/lemmy) repo.

### Summary

Unblocking an admin of an instance is not possible. When going to the admin's profile and then pressing unblock, the message "cant_block_admin" is displayed.

### Steps to Reproduce

This is hard to reproduce because banning an admin is not possible anymore. This issue only affects users who blocked an admin before the update that disallowed this process was introduced.

@dessalines if you see this. I did not block you on purpose, this happened accidentally and I would like to reverse this.

### Technical Details

<img width="894" alt="Screenshot 2023-06-22 at 22 10 11" src="https://github.com/LemmyNet/lemmy-ui/assets/13732136/9cacfcd7-b021-4c75-9ae3-eff57f314532">

### Version

BE: 0.18.0-rc.6

### Lemmy Instance URL

lemmy.ml

**Comment by Tuncay-Ayhan** [comment]
> ### Requirements
> * [x]  Is this a bug report? For questions or discussions use https://lemmy.ml/c/lemmy_support
> * [x]  Did you check to see if this issue already exists?
> * [x]  Is this only a single bug? Do not put multiple bugs in one issue.
> * [ ]  Is this a server side (not related to the UI) issue? Use the [Lemmy back end](https://github.com/LemmyNet/lemmy) repo.
> 
> ### Summary
> Unblocking an admin of an instance is not possible. When going to the admin's profile and then pressing unblock, the message "cant_block_admin" is displayed.
> 
> ### Steps to Reproduce
> This is hard to reproduce because banning an admin is not possible anymore. This issue only affects users who blocked an admin before the update that disallowed this process was introduced.
> 
> @dessalines if you see this. I did not block you on purpose, this happened accidentally and I would like to reverse this.
> 
> ### Technical Details
> <img alt="Screenshot 2023-06-22 at 22 10 11" width="894" src="https://user-images.githubusercontent.com/13732136/248049555-9cacfcd7-b021-4c75-9ae3-eff57f314532.png">
> ### Version
> BE: 0.18.0-rc.6
> 
> ### Lemmy Instance URL
> lemmy.ml

Maybe this will sound too obvious, but did you try "Unblock user" at the top right corner?

**Comment by phinas** [comment]
Yes, that is the issue. If I press "Unblock user", the message "cant_block_admin" is being shown, which you can also see in the bottom left corner on the screenshot.

**Comment by Tuncay-Ayhan** [comment]
> Yes, that is the issue. If I press "Unblock user", the message "cant_block_admin" is being shown, which you can also see in the bottom left corner on the screenshot.

Gotcha. Didn't notice it was a message instead of a button. I thought that was the button you were pressing on 🙃

**Comment by Nutomic** [comment]
You can unblock through the database, not worth adding extra logic to handle this case.
